community/radare2.hpkg

```diff
@@ -0,0 +1,92 @@
+(use ../prelude)
+(import ../core)
+(use ./capstone)
+(use ./file)
+(use ./xxhash)
+(use ./zlib)
+(use ./libzip)
+(use ./lz4)
+(use ./openssl)
+(use ./libuv)
+
+# XXX tree-sitter can be used to enable features.
+
+(defsrc radare2-src
+  :file-name
+  "radare2-4.4.0.tar.gz"
+  :url
+  "https://github.com/radareorg/radare2/archive/4.4.0.tar.gz"
+  :hash
+  "sha256:401ab89f39f7e70e03336f6890dc4fdf52dae4f61201e4cd8a1fbe5a4b9eb451")
+
+(defpkg radare2
+  :builder
+  (fn []
+    (os/setenv "PKG_CONFIG_PATH"
+               (string
+                (join-pkg-paths ":" "/lib64/pkgconfig"
+                                [libuv libzip])
+                ":"
+                (join-pkg-paths ":" "/lib/pkgconfig"
+                                [capstone zlib openssl xxhash lz4])))
+
+    (os/setenv "PATH"
+               (join-pkg-paths ":" "/bin"
+                               [core/coreutils
+                                core/findutils
+                                core/gcc
+                                core/grep
+                                core/make
+                                core/patch
+                                core/sed
+                                core/awk
+                                core/tar
+                                core/pkgconf]))
+    (core/link-/bin/sh)
+    (unpack-src radare2-src)
+
+    (os/setenv "CFLAGS"
+               (string *default-cflags*
+                       " "
+                       (string "-I" (file :path) "/include")
+                       " "
+                       # capstone does this weird thing that there is a `capstone` directory in the
```

Comment:
I'm not sure if this long comment here is warranted. Should I remove it?

Reply:
I think it's not bad to document this type of thing.

Reply:
I think I encountered a similar situation for ncurses and possibly other things.

I haven't convinced myself why the issue exists yet -- may be there is something I could do to get things working appropriately.

May be we could name such situations (e.g. unaligned include path or something), then the comment wouldn't have to be long.  Though perhaps we'd want some place for a reader to be able to look to learn what the name means.

Reply:
If we take notes / log about situations we encountered, perhaps these can be organized and referred to.  This ought to help us in future work possibly also for potential contributors.

I was thinking that the topic for a package in the intent-to-package stream might not be a bad place to take some notes.

Reply:
I'm fine with the long comment for now.  I think it can be shortened or removed later.

Reply:
I noted this under the `capstone` package in the `intent-to-package` stream.